ospatch/apt_upgrade.go

```diff
@@ -15,32 +15,15 @@
 package ospatch
 
 import (
-	"fmt"
-	"os"
 	"os/exec"
-)
-
-const aptGet = "/usr/bin/apt-get"
-
-var (
-	aptGetUpdateArgs      = []string{"update"}
-	aptGetUpgradeArgs     = []string{"upgrade", "-y"}
-	aptGetFullUpgradeArgs = []string{"full-upgrade", "-y"}
-	aptGetDistUpgradeArgs = []string{"dist-upgrade", "-y"}
-)
 
-type aptGetUpgradeType int
-
-const (
-	aptGetUpgrade aptGetUpgradeType = iota
-	// AptGetDistUpgrade specifies apt-get dist-upgrade should be run.
-	AptGetDistUpgrade
-	// AptGetFullUpgrade specifies apt-get full-upgrade should be run.
-	AptGetFullUpgrade
+	"github.com/GoogleCloudPlatform/guest-logging-go/logger"
+	"github.com/GoogleCloudPlatform/osconfig/inventory/packages"
 )
 
 type aptGetUpgradeOpts struct {
-	upgradeType aptGetUpgradeType
+	upgradeType packages.AptUpgradeType
+	includes    []string
```

Comment:
we finalized on the api field names and decided to use "exclusive-(packages|patches)". exclusive_patches for windows and zypper and exclusive_packages for apt and yum